website/content/docs/install/production/requirements.mdx

```diff
@@ -108,30 +108,87 @@ net.bridge.bridge-nf-call-ip6tables = 1
 net.bridge.bridge-nf-call-iptables = 1
 ```
 
-## User Permissions
+## Hardening Nomad
+
+As noted in the [Security Model][] guide, Nomad is not **secure-by-default**.
+
+### User Permissions
 
 Nomad servers and Nomad clients have different requirements for permissions.
 
 Nomad servers should be run with the lowest possible permissions. They need
 access to their own data directory and the ability to bind to their ports. You
-should create a `nomad` user with the minimal set of required privileges.
+should create a `nomad` user with the minimal set of required privileges. If you
+are installing Nomad from the official Linux packages, the systemd unit file
+runs Nomad as `root`. For your server nodes you should change this to a
+minimally privileged `nomad` user.
 
 Nomad clients must be run as `root` due to the OS isolation mechanisms that
-require root privileges (see also [Linux Capabilities] below). The Nomad
+require root privileges (see also [Linux Capabilities][] below). The Nomad
 client's data directory should be owned by `root` with filesystem permissions
 set to `0700`.
 
-## Linux Capabilities
+### Linux Capabilities
 
 On Linux, Nomad clients require privileged capabilities for isolating
 tasks. Nomad clients require `CAP_SYS_ADMIN` for creating the tmpfs used for
-secrets, bind-mounting task directories, mounting volumes, and running some
-task driver engines. Nomad clients require `CAP_NET_ADMIN` for a variety of
-tasks to set up networking. You should run Nomad as `root`, but running as
-`root` does not grant these required capabilities if Nomad is running in a
-user namespace. Running Nomad clients inside a user namespace is
-unsupported. See the [`capabilities(7)`] man page for details on Linux
-capabilities.
-
+secrets, bind-mounting task directories, mounting volumes, and running some task
+driver engines. Nomad clients require `CAP_NET_ADMIN` for a variety of tasks to
```

Comment:
```suggestion
driver plugins. Nomad clients require `CAP_NET_ADMIN` for a variety of tasks to
```

we don't use "engine" anywhere else

Reply:
In this case I'm referring to "the thing the task driver plugin talks to". Ex. `dockerd` as opposed to the `docker` plugin. We probably need to come up with some kind of vocabulary around that. But for now I can clean this sentence up to avoid it.